defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+configure_kernel: true
```

Comment:
Maybe I would have those variables called `{name}_enabled: true/false`. But up to you.